index.php

```diff
@@ -40,6 +40,12 @@
     }
 }
 
+if (!file_exists('vendor/autoload.php')) {
+    echo '<h2>Error</h2>';
+    echo 'Please execute "composer install" from the command line to install the required dependencies for Bolt.';
+    return false;
+}
+
```

Comment:
Not always valid. If someone has installed via an archive and for some reason that file doesn't exist they would get a confusing error.